config/Shared/config_default.php

```diff
@@ -810,6 +811,7 @@
     ReadyForMerchantAppOnboardingTransfer::class => 'merchant-app-events',
     MerchantAppOnboardingStatusChangedTransfer::class => 'merchant-app-events',
     AppConfigUpdatedTransfer::class => 'app-events',
+    PaymentUpdatedTransfer::class => 'payment-events',
```

Comment:
please move to 
` PaymentCreatedTransfer::class => 'payment-events',`
to have them grouped